tests/python_tests/test_generation_config.py

```diff
@@ -7,11 +7,21 @@
 import os
 import pytest
 
+
+def verify_set_values(generation_config, kwargs):
+    generation_config.validate()
+    for key, value in kwargs.items():
+        assert getattr(generation_config, key) == value
```

Comment:
will it work when kargs have different values for eos and stop_token_ids ? stop_token_ids will be different than original one